styles/variables-base.less

```diff
@@ -344,47 +344,59 @@
-@agate-input-small-font-size: 24px;
-@agate-input-border-width: 1px;
+@agate-input-decorator-border-bottom-width: 0;
 @agate-input-decorator-border-width: 1px;
 @agate-input-decorator-focus-border-width: 4px;
+@agate-input-decorator-has-value-opacity: inherit;
 @agate-input-decorator-height: inherit;
+@agate-input-decorator-opacity: 1;
+@agate-input-disabled-opacity: @agate-disabled-opacity;
 @agate-input-focused-opacity: inherit;
+@agate-input-font-size: 30px;
+@agate-input-font-weight: @agate-base-font-weight;
+@agate-input-has-value-opacity: @agate-input-decorator-opacity;
+@agate-input-hasiconafter-padding-right: inherit;
+@agate-input-hasiconbefore-padding-left: inherit;
+@agate-input-height: (@agate-icon-size - ((@agate-input-decorator-border-width * 2) + (extract(@agate-input-padding, 2) * 2)));
+@agate-input-icon-after-margin: 0 (@agate-spotlight-outset / 2) 0 0;
+@agate-input-icon-after-padding-left: 15px;
+@agate-input-icon-after-padding-right: 15px;
+@agate-input-icon-after-padding-top-bottom: inherit;
+@agate-input-icon-align: inherit;
+@agate-input-icon-before-margin: 0 0 0 (@agate-spotlight-outset / 2);
+@agate-input-icon-before-padding-left: 15px;
+@agate-input-icon-before-padding-right: 15px;
+@agate-input-icon-before-padding-top-bottom: inherit;
+@agate-input-icon-display: inherit;
+@agate-input-icon-empty-value-focused-opacity: @agate-input-focused-opacity;
+@agate-input-icon-opacity: @agate-input-decorator-opacity;
+@agate-input-large-icon-after-padding-left: @agate-input-icon-after-padding-left;
+@agate-input-large-icon-before-padding-right: @agate-input-icon-before-padding-right;
+@agate-input-large-icon-font-size: 96px;
+@agate-input-large-icon-height: @agate-icon-size;
 @agate-input-margin: @agate-input-decorator-focus-border-width (@agate-spotlight-outset / 2);
+@agate-input-max-width: none;
+@agate-input-min-width: 0;
 @agate-input-opacity: inherit;
-@agate-input-padding: 21px 32px;
-@agate-input-padding-top-bottom: 21px;
 @agate-input-padding-left: 32px;
 @agate-input-padding-right: 32px;
-@agate-input-icon-display: inherit;
-@agate-input-icon-align: inherit;
-@agate-input-icon-after-padding-top-bottom: inherit;
-@agate-input-icon-after-padding-right: 15px;
-@agate-input-icon-after-padding-left: 15px;
-@agate-input-icon-before-padding-top-bottom: inherit;
-@agate-input-icon-before-padding-right: 15px;
-@agate-input-icon-before-padding-left: 15px;
-@agate-input-icon-before-margin: 0 0 0 (@agate-spotlight-outset / 2);
-@agate-input-icon-after-margin: 0 (@agate-spotlight-outset / 2) 0 0;
-@agate-input-hasiconbefore-padding-left: inherit;
-@agate-input-hasiconafter-padding-right: inherit;
-@agate-input-height: (@agate-icon-size - ((@agate-input-border-width * 2) + (extract(@agate-input-padding, 2) * 2)));
-@agate-input-small-height: (@agate-button-small-height - (@agate-input-border-width * 2));
-@agate-input-text-align: inherit;
+@agate-input-padding-top-bottom: 21px;
+@agate-input-padding: 21px 32px;
```

Comment:
@agate-input-padding-lef

Reply:
@agate-input-padding is used to set @agate-input-height, so by removing it we would have to change the implementation and that would also affect other skins.